packages/connect/src/device/Device.ts

```diff
@@ -773,8 +773,9 @@ export class Device extends TypedEmitter<DeviceEvents> {
 
         const btcOnly = this.firmwareType === FirmwareType.BitcoinOnly;
         const binary = await getBinaryOptional({ baseUrl, btcOnly, release });
-        if (binary === null) {
+        if (binary === null || binary.byteLength < 200) {
```

Comment:
I would personally be more explicit, something like 

```
if (binary.byteLenght < 200) {
  log('skipping too small);
...
}
```

otherwise 👍

Reply:
Done :heavy_check_mark: